### PR TITLE
Package and cut Releases in CI

### PR DESCRIPTION
**Pull request by JonathanPorta**
- Help ensure that we only release working, tested, build artifacts.
- Decrease the management overhead (nightmare) of building in production.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,3 +22,14 @@ npm install # The command shouldn't fail.
 npm test # The tests should pass, and the command shouldn't fail.
 npm start # The server will come up on localhost:8080, and print a banner.
 ```
+## Deployment
+In order to cut a new release, you must bump the version in `package.json`. Any merge into `master` that successfully passes testing in CI and has a bumped version in `package.json` will be packaged into an RPM and pushed to the [mtna package cloud repository][repo_url].
+
+## Installation in Production
+To make installation easier an install script is available too bootstrap your system. The script will add the [mtna package cloud repository][repo_url] to your RPM configuration and then install the mtna package.
```
Comment (DavidSouther) [inline README.md]: "is available too bootstrap your system" Spelling - s/too/to/

File: package.sh [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+project=$(cat package.json | jq -r .name)
+version=$(cat package.json | jq -r .version)
+iteration="$TRAVIS_BUILD_NUMBER.git$(git rev-parse --short HEAD)"
+architecture='x86_64'
+url="https://github.com/PronghornDigital/Montana-News-Archive"
+vendor=$(cat package.json | jq -r .author)
+description=$(cat package.json | jq -r .description)
+install_prefix="/opt/PronghornDigital/${project}"
+
+echo "Building RPM..."
```
Comment (DavidSouther) [inline package.sh]: heredocs for this blob

File: package.sh [hunk 3]
```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+project=$(cat package.json | jq -r .name)
+version=$(cat package.json | jq -r .version)
+iteration="$TRAVIS_BUILD_NUMBER.git$(git rev-parse --short HEAD)"
```
Comment (DavidSouther) [inline package.sh]: Travis_BUILD_NUMBER should have a default value for when running locally as testing

Comment (JonathanPorta) [inline package.sh]: Yeah, but, I don't really want someone to push a bunch of packages from local. I want that controlled by CI.

I'll add a guard statement at the top that exits the script if travis_build_number is not defined.

**Comment by DavidSouther** [comment]
Lgtm

**Pull request merged**